package/MDAnalysis/coordinates/DLPoly.py

```diff
@@ -168,9 +168,9 @@ def _read_next_timestep(self, ts=None):
         if not line.startswith('timestep'):
             raise IOError
         if not self._imcon == 0:
-            ts._unitcell[0] = map(float, self._file.readline().split())
-            ts._unitcell[1] = map(float, self._file.readline().split())
-            ts._unitcell[2] = map(float, self._file.readline().split())
+            ts._unitcell[0] = list(map(float, self._file.readline().split()))
```

Comment:
shouldn't that be better replaced with `[float(val) for val in self._file.readline().split()]`?